sdarq/backend/src/app.py

```diff
@@ -226,6 +228,9 @@ def submit_app():
     try:
         json_data = request.get_json()
         dojo_name = json_data['Service']
+        regex = "^[a-zA-Z0-9][a-zA-Z0-9-_ ]{1,40}[a-zA-Z0-9]$"
+        if not (re.search(regex,dojo_name)):
```

Comment:
add logging.error here, or remove from line 172?

Reply:
the value error that is on the next line gets logged.